src/git/install.sh

```diff
@@ -212,12 +242,20 @@ elif [ "${ADJUSTED_ID}" = "rhel" ]; then
     if [ $ID = "mariner" ]; then
         check_packages glibc-devel kernel-headers binutils
     fi
+
+else
+    echo "Linux distro ${ID} not supported."
+    exit 1
 fi
 
 # Partial version matching
 if [ "$(echo "${GIT_VERSION}" | grep -o '\.' | wc -l)" != "2" ]; then
     requested_version="${GIT_VERSION}"
-    version_list="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/git/git/tags" | grep -oP '"name":\s*"v\K[0-9]+\.[0-9]+\.[0-9]+"' | tr -d '"' | sort -rV )"
+    if [ "${ADJUSTED_ID}" = "alpine" ]; then
+        version_list="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/git/git/tags" | jq -r '.[] | .name' | grep -E '^v[0-9]+\.[0-9]+\.[0-9]+$' | sed 's/^v//g' | sort -rV )"
+    else
+        version_list="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/git/git/tags" | grep -oP '"name":\s*"v\K[0-9]+\.[0-9]+\.[0-9]+"' | tr -d '"' | sort -rV )"
+    fi
```

Comment:
```suggestion
version_list="$(curl -sSL -H "Accept: application/vnd.github.v3+json" "https://api.github.com/repos/git/git/tags" | jq -r '.[] | .name' | grep -E '^v[0-9]+\.[0-9]+\.[0-9]+$' | sed 's/^v//g' | sort -rV )"
```

(untested)

Reply:
Reverted to PCRE grep implementation (figured out how to support it in Alpine)